### ISSUE TITLE
Deviation from Spec?: Transports attribute 

### ISSUE THREAD

**Issue by jjjacksn** [duo-labs/py_webauthn#149]
Thanks for the great project! It's been great to use getting started with WebAuthn in python.

However, I believe I have discovered a deviation from the WebAuthn spec in the struct helpers.

I discovered this while implementing a python backend using py-webauthn and a front end using [SimpleWebAuthn](https://simplewebauthn.dev/).

py_webauthn's [`RegistrationCredential`](https://github.com/duo-labs/py_webauthn/blob/master/webauthn/helpers/structs.py#L370) has a top level `transports` attribute while SimpleWebAuthn has it at the `response.transports` key path level in the [AuthenticatorAttestationResponseJSON](https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/typescript-types/src/index.ts#L128) interface.

From my quick reading of the [WebAuthn 2.0 spec](https://www.w3.org/TR/webauthn-2/) , it seems that having this data at `response.transports` is correct.

**Comment by MasterKale** [comment]
Hello @jjjacksn, thanks for calling this out. Way back when I defined `transports` in the same place in **SimpleWebAuthn** (in `PublicKeyCredential` instead of within `response`), but I recently moved `transports` into `response` as part of updating that library to accommodate the JSON data structures I got into the WebAuthn L3 draft [here](https://w3c.github.io/webauthn/#dictdef-authenticatorattestationresponsejson). Updating **py_webauthn** accordingly (which will include relocating `transports` into `response` too) will happen when I address #143.
